products/views.py

```diff
@@ -0,0 +1,72 @@
+import random
+
+from django.views import View
+from django.http import JsonResponse
+
+from . import models
+from reviews import models as reviews_models
```

Comment:
```suggestion
from reviews.models import Review
```
사용할 클래스를 바로 import 해주세요